cmd/kubelet/app/options/options.go

```diff
@@ -189,8 +189,11 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.RunOnce, "runonce", s.RunOnce, "If true, exit after spawning pods from local manifests or remote urls. Exclusive with --api-servers, and --enable-server")
 	fs.BoolVar(&s.EnableDebuggingHandlers, "enable-debugging-handlers", s.EnableDebuggingHandlers, "Enables server endpoints for log collection and local running of containers and commands")
 	fs.DurationVar(&s.MinimumGCAge.Duration, "minimum-container-ttl-duration", s.MinimumGCAge.Duration, "Minimum age for a finished container before it is garbage collected.  Examples: '300ms', '10s' or '2h45m'")
+	fs.MarkDeprecated("minimum-container-ttl-duration", "will be removed in a future version")
```

Comment:
Yeah. The eviction flags should be suggested as alternatives.

On Wed, Jun 22, 2016 at 11:38 AM, ronnielai <notifications@github.com>
wrote:

> In cmd/kubelet/app/options/options.go
> <https://github.com/kubernetes/kubernetes/pull/27881#discussion_r68110138>
> :
>
> > @@ -189,8 +189,11 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
> >  	fs.BoolVar(&s.RunOnce, "runonce", s.RunOnce, "If true, exit after spawning pods from local manifests or remote urls. Exclusive with --api-servers, and --enable-server")
> >  	fs.BoolVar(&s.EnableDebuggingHandlers, "enable-debugging-handlers", s.EnableDebuggingHandlers, "Enables server endpoints for log collection and local running of containers and commands")
> >  	fs.DurationVar(&s.MinimumGCAge.Duration, "minimum-container-ttl-duration", s.MinimumGCAge.Duration, "Minimum age for a finished container before it is garbage collected.  Examples: '300ms', '10s' or '2h45m'")
> > +	fs.MarkDeprecated("minimum-container-ttl-duration", "will be removed in a future version")
>
> The eviction proposal
> <https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/kubelet-eviction.md#deprecation-of-existing-features>
> deprecates these 3 flags without introducing any new flags for replacement.
> Do you mean the eviction-related flags should be mentioned here?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27881/files/31c2c066f8680dfcb41b7d0223066c544188a87d#r68110138>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGvIKH3lYyE48XbhBNGCRPihEtVy84lQks5qOYESgaJpZM4I8AQH>
> .
>